src/components/ContentSidebar/SidebarFileProperties.js

```diff
@@ -55,7 +57,7 @@ const SidebarFileProperties = ({
     isLoading,
     intl,
 }: Props) => {
-    const value = getProp(file, `${FIELD_METADATA_CLASSIFICATION}.${KEY_CLASSIFICATION_TYPE}`);
+    const value = getProp(classification, KEY_CLASSIFICATION_TYPE);
```

Comment:
probably a better variable name other than value

Reply:
renamed to `classificationName`